CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.4.0
+
```

Comment:
typically we provide the matching sdk version here. see below for examples

Reply:
done